benchmarks/AuraDiBench.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace PhpBench\Benchmarks\Container;
+
+use Aura\Di\ContainerBuilder;
+
+/**
+ * @Groups({"aura-di"}, extend=true)
+ */
+class AuraDiBench extends ContainerBenchCase
+{
+    private $container;
+
+    public function benchGetUnoptimized()
```

Comment:
I have skipped this benchmark containers that don't "optimize" (not a very good term, basically containers that don't do any definition caching).

```php
/**
 * @Skip()
 */
public function benchGetUnoptimized()
{
}
```